### ISSUE TITLE
Slim VGG losses increase gradually with default training configuration

### ISSUE THREAD

**Issue by kakpple** [tensorflow/tensorflow#15939]
Hi, when I try to train imagenet with slim vgg network with default configuration,
The loss increases gradually from ~0.1 to over 10000. 
I am not even able to debug this issue because, all tensors losses are encapsulated inside slim.
Is there any way to debug this issue?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by kakpple** [comment]
I find out that there was something wrong with the imagenet tfrecord.
Actually, in tfrecord image was labeled beyond 1000, larger than 1000.
I remade the tfrecord, and mobilenet and resnet is learning.
Anyway, thank you~

**Comment by reedwm** [comment]
Closing since the issue is resolved.
